x-pack/test/functional/apps/transform/index.ts

```diff
@@ -36,16 +36,16 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       await transform.testResources.resetKibanaTimeZone();
     });
 
-    loadTestFile(require.resolve('./permissions'));
+    // loadTestFile(require.resolve('./permissions'));
```

Comment:
Don't forget to re-enable these tests back before merging

Reply:
Thanks for catching! Fixed it in [c64f1b2](https://github.com/elastic/kibana/pull/149049/commits/c64f1b2b2c43f596b6a4adbead846a4d6e2a56f8). (I think with the latest merge of `main` into this branch that whole code part was refactored away though, it's now organized differently)